runtime/doc/msgpack_rpc.txt

```diff
@@ -413,5 +417,30 @@ of update.
 ["popupmenu_hide"]
 	The popupmenu is hidden.
 
+["cmdline_enter"]
+	Enter the cmdline.
+
+["cmdline_leave"]
+	Leave the cmdline.
```

Comment:
I would argue they are not synonymous. If we get `mode_change` to support cmdline, it should indicate cmdline regardless of `cmdline_external`.  However these events are emitted only when `cmdline_external`, and the ui thus should draw the cmdline, though they should be called `show` and `hide` in consistency with popupmenu.